packages/client-core/src/admin/components/locations/AddEditLocationModal.tsx

```diff
@@ -97,7 +98,12 @@ export default function AddEditLocationModal(props: {
   const screenSharingEnabled = useHookstate<boolean>(location?.locationSetting.screenSharingEnabled || true)
   const locationType = useHookstate(location?.locationSetting.locationType || 'public')
   const sceneThumbnailState = useHookstate(getMutableState(SceneThumbnailState))
-  const sceneSettingsEntities = useQuery([SceneSettingsComponent])
+  // only use query if engine is running
+  let sceneSettingsEntities: Entity[] = []
+  if (getState(EngineState).viewerEntity !== UndefinedEntity) {
+    // no viewerEntity if not in Editor, running the engine
+    sceneSettingsEntities = useQuery([SceneSettingsComponent])
```

Comment:
We should put the conditional logic after this, react requires the same number of hooks across renders